src/io/flutter/preview/PreviewView.java

```diff
@@ -182,8 +173,23 @@ public void onResponse(FlutterOutline widget, JsonObject response) {
     }
 
     @Override
-    public void onFailure(Throwable e) {
-      noWidget();
+    public void onFailure(RenderProblemKind kind) {
+      ApplicationManager.getApplication().invokeLater(() -> {
+        if (previewArea != null) {
+          switch (kind) {
+            case EXCEPTION:
+            case NO_TEMPORARY_DIRECTORY:
+              previewArea.clear("There was an exception during rendering");
```

Comment:
Should we encourage them to file a bug report here?  Maybe a link to submit feedback?

Reply:
In theory these should never happen.

Reply:
👍